src/app/app.component.html

```diff
@@ -1,2 +1,2 @@
 <h1>NgWig Demo</h1>
-<ng2wig></ng2wig>
+<ng2wig plugins="demo"></ng2wig>
```

Comment:
ok, the question is still actual, so I recreated it and tried to make it more specific
https://stackoverflow.com/questions/44654508/angular-creating-plugins-for-3rd-party-packages-libraries